examples/demo.js

```diff
@@ -146,22 +151,22 @@ window.onload = function () {
           let modal = $('#js-download-gif-modal');
 
           $('#js-download-as-gif-button').one('click', function () {
-            // Trigger download
+            // Trigger download.
             downloadGif(image);
-            // Close modal
+            // Close modal.
             modal.modal('hide');
           });
 
           var gifContainer = document.getElementById('js-download-modal-gif-container');
 
-          // Clear previous results
+          // Clear previous results.
           gifContainer.innerHTML = '';
 
-          // Insert image
+          // Insert image.
           gifContainer.appendChild(animatedImage);
 
```

Comment:
Why do we have multiple line spaces here?
I know it is not in your changes in this pr but curious to know the reason.

Reply:
Probably some code was deleted in between PRs. This old code needs to be fixed. I had suggested a new UI some months back https://github.com/publiclab/image-sequencer/issues/1238 because the existing one is too messy. I have already started work on it.